gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFileSystem.java

```diff
@@ -1008,11 +1003,11 @@ protected int getDefaultPort() {
 
   @Override
   public boolean hasPathCapability(Path path, String capability) {
-    switch (validatePathCapabilityArgs(path, capability)) {
-        // TODO: remove string literals in favor of Constants in CommonPathCapabilities.java
-        // from Hadoop 3 when Hadoop 2 is no longer supported
-      case "fs.capability.paths.append":
-      case "fs.capability.paths.concat":
+    checkNotNull(path, "path must not be null");
+    checkArgument(!isNullOrEmpty(capability), "capability must not be null or empty string");
```

Comment:
nit: Consider adding the path information in the error message.

Reply:
Done.